assets/js/components/Root/index.js

```diff
@@ -36,21 +41,25 @@ import { Provider as ViewContextProvider } from './ViewContextContext';
 import InViewProvider from '../InViewProvider';
 
 export default function Root( { children, registry, viewContext = null } ) {
+	const inViewState = useRef( {
```

Comment:
Only because this one never changes, so I didn't think it was needed to create the state variable. But since we use state for this elsewhere: happy to do so 👍🏻